src/connection_string.ts

```diff
@@ -1292,5 +1292,6 @@ export const DEFAULT_OPTIONS = new CaseInsensitiveMap(
  */
 export const FEATURE_FLAGS = new Set([
   Symbol.for('@@mdb.skipPingOnConnect'),
-  Symbol.for('@@mdb.enableMongoLogger')
+  Symbol.for('@@mdb.enableMongoLogger'),
+  Symbol.for('@@mdb.internalLoggerConfig')
```

Comment:
This may not be the correct place for this, so I'm open to moving the logic to accept the internal config elsewhere in this file/the `parseOptions` function

Reply:
LGTM, we need a way to get our settings in there for testing, as long as we capture this information. Can we add some basic unit tests to: `test/integration/node-specific/feature_flags.test.ts`. I'd like something that makes sure we don't drop things from here unintentionally thinking it's "unused".